full-backend-tests/src/test/java/org/graylog/datanode/RemoteReindexingMigrationIT.java

```diff
@@ -35,7 +35,7 @@
 import org.graylog.testing.containermatrix.SearchServer;
 import org.graylog.testing.containermatrix.annotations.ContainerMatrixTest;
 import org.graylog.testing.containermatrix.annotations.ContainerMatrixTestsConfiguration;
-import org.hamcrest.Matchers;
+import org.jetbrains.annotations.NotNull;
```

Comment:
ide reference shouldn't be in here, I think

Reply:
Good point, fixed